### DISCUSSION TITLE
SSH

### DISCUSSION THREAD

**Discussion by marc31** [lhns/restic-android#183] [Ideas]
I am thinking about add support for ssh.
Do you plan to do this ? 
Do you have an idea if is doable ?
I was thinking about donwload ssh from termux and do some stuff is it the right way ?

**Comment by lhns** [comment]
Hi this could work well and might be a good template for rclone support in the future. Might require some experimentation though.

**Comment by marc31** [comment]
Before trying out your application, I directly tested Restic using Termux. I wanted to mount backups, but for that to work, it's necessary to have a rooted phone (because access to /dev/ is locked). Then, I came across several forums discussing Rclone, stating that it's impossible without a rooted phone. but I'm not sure anymore if users were only talking about Rclone's mounting function.

**Reply by veryniceworld** [reply]
`rclone` support is very useful for backup to other cloud storage such as Nextcloud, Pcloud, MegaIO ... etc.   I have using using `restic` with `rclone` under _termux_ for years.  Looking forward to your app's `rclone` suport.
